### PR TITLE
Create Analyzing a transmon and a resonator.ipynb

### LINKED ISSUE #210: Ansys tutorial: compare freq of resonator with and without qubit
### What is the feature being requested?

Tutorial / ppt on freq of readout for
1) just CPW 
2) CPW loaded by qubit 
analyzed with eigenmode ansys 

image

image

### PR DESCRIPTION
**Pull request by smukesh25**
### What are the issues this pull addresses (issue numbers / links)?
Closes #210 

### Did you add tests to cover your changes (yes/no)?
Yes

### Did you update the documentation accordingly (yes/no)?
Yes

### Did you read the CONTRIBUTING document (yes/no)?
Yes

### Summary
Created a notebook showing a transmon with one CPW resonator connected to it and different analyses on this circuit.

### CODE REVIEW

**Review by priti-ashvin-shah-ibm — APPROVED** [review]
LGTM, ran it on Crum.

**Review by JeremyDrysdale-ibm — DISMISSED** [review]
LGTM

File: tutorials/6 Analysis/Analyzing a transmon and a resonator.ipynb [hunk 1]
```diff
@@ -0,0 +1,654 @@
...
+   "source": [
+    "hfss.plot_ansys_delete(['Mag_E1'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "olive-publicity",
+   "metadata": {},
+   "source": [
+    "To look at the second mode created, we use the following command, and then plot the corresponding E-field."
```
Comment (ThomasGM4) [inline tutorials/6 Analysis/Analyzing a transmon and a resonator.ipynb:293]: maybe "To look at the second eigenmode,..."

File: tutorials/6 Analysis/Analyzing a transmon and a resonator.ipynb [hunk 2]
```diff
@@ -0,0 +1,654 @@
...
+    "hfss.plot_ansys_fields('main')\n",
+    "hfss.save_screenshot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "negative-birthday",
+   "metadata": {},
+   "source": [
+    "We delete this design to prepare of further analysis."
```
Comment (ThomasGM4) [inline tutorials/6 Analysis/Analyzing a transmon and a resonator.ipynb:323]: "We delete this design to prepare **for** further analysis."

File: tutorials/6 Analysis/Analyzing a transmon and a resonator.ipynb [hunk 3]
```diff
@@ -0,0 +1,654 @@
...
+    "q3d.connect_ansys()\n",
+    "q3d.activate_q3d_design(\"TransmonResonator_q3d\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "caroline-belle",
+   "metadata": {},
+   "source": [
+    "Next, we render the exisitng design to Ansys Q3D for analysis. To ensure that the readout is insulated from the ground plane, we specify the 'open' attribute of render_design to be 'readout'."
```
Comment (ThomasGM4) [inline tutorials/6 Analysis/Analyzing a transmon and a resonator.ipynb:502]: "... we specify the 'open' attribute of render_design to be 'readout'."  ->  "... we set the `readout` pin of Q1  to have an open termination."

File: tutorials/6 Analysis/Analyzing a transmon and a resonator.ipynb [hunk 4]
```diff
@@ -0,0 +1,654 @@
...
+    "Using capacitance matrices obtained from each pass, save the many parameters of the Hamiltonian of the system. lumped_oscillator_vs_passes takes in the following parameters: <br><br>\n",
+    "Lj_nH: float <br>\n",
+    "Cj_fF: float <br>\n",
+    "N: int <br>\n",
+    "fr: Union[list, float] <br>\n",
+    "fb: Union[list, float] <br>\n",
+    "maxPass: int <br>\n",
+    "variation: str = '' <br>\n",
+    "solution_kind: str = 'AdaptivePass' <br>\n",
+    "g_scale: float = 1"
```
Comment (ThomasGM4) [inline tutorials/6 Analysis/Analyzing a transmon and a resonator.ipynb:590]: If want, can expand the explanation of the inputs;

    Lj - the Josephson inductance of your Josephson junction 
    Cj - the capacitance of your Josephson junction
    N - the total number of connection pads 
    fr - the frequency of the readout resonator 
    [fb1, fb2, fb3...., fbN-1] - list of the frequencies of the busses
    passes - the number of passes your simulation ran for

**Review by ThomasGM4 — COMMENTED** [review]
Some minor suggested changes, but looks good!

**Review by priti-ashvin-shah-ibm — APPROVED** [review]
LGTM

**Review by ThomasGM4 — APPROVED** [review]

**Review by JeremyDrysdale-ibm — APPROVED** [review]
LGTM

**Pull request merged**